bin/get_annotations.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+import click
+import pdfrw
+
+
+@click.command()
+@click.argument("template")
+def get_annotations(template):
+    """
+    This utility prints all the annotations in the provided template.
+        template: A path to the template that you would like to get annotations for.
+    """
+
+    pdf = pdfrw.PdfReader(template)
+    annotations = pdf.pages[0]["/Annots"]
+    print("Here is a list of annotations in the document.\n")
+    try:
+        while True:
+            annotation = annotations.pop()
```

Comment:
Why are you using a `while True` loop and catching an `IndexError` instead of iterating over the keys using `for annotation in annotations.values()`?

Reply:
It doesn't have keys or values or anything useful like that unfortunately. It just has pop.

Reply:
`pop()` in this case is an array method. I ran the command locally and confirmed that `annotations` is an array. Maybe there was a typo when you tried last time?